source/administration/object-management/object-lifecycle-management.rst

```diff
@@ -10,41 +10,46 @@ Object Lifecycle Management
    :local:
    :depth: 1
 
-MinIO Object Lifecycle Management allows creating rules for time or date
-based automatic transition or expiry of objects. For object transition,
-MinIO automatically moves the object to a configured remote storage
-tier. For object expiry, MinIO automatically deletes the object.
+MinIO Object Lifecycle Management allows creating rules for time or date based automatic transition or expiry of objects. 
+For object transition, MinIO automatically moves the object to a configured remote storage tier. 
+For object expiry, MinIO automatically deletes the object.
 
-MinIO lifecycle management is built for behavior and syntax compatibility with 
-:s3-docs:`AWS S3 Lifecycle Management <object-lifecycle-mgmt.html>`. For
-example, you can export S3 lifecycle management rules and import them into
-MinIO or vice-versa. MinIO uses JSON to describe lifecycle management rules,
-and conversion to or from XML may be required. 
+MinIO lifecycle management is built for behavior and syntax compatibility with :s3-docs:`AWS S3 Lifecycle Management <object-lifecycle-mgmt.html>`. 
```

Comment:
Maybe `provides` instead of `is built for` to avoid the passive?

Reply:
Did a rewrite here.